hovercraft/local_directives/example.__init__.py

```diff
@@ -0,0 +1,253 @@
+"""
+This is an example directive. To use this as a template for your own directive, create a new directory within the local_directives directory. Next move this file into the new directory and reanme it to __init__.py. While this file defines a video directive, it can be used as a skeleton to build any sort of directive you might need.
```

Comment:
Please use a maximum line length of around 80 characters. The diff tool won't wrap lines for display, so I can't read them for review, that's the only reason.